CONTRIBUTING_MODELS.md

```diff
@@ -72,168 +119,174 @@ contribution! 🙂
+- Add a task-specific head (e.g., classifier head, LM head)
 
-#### Convert weights from the original source and check output!
+##### API
 
-Before you open a PR for adding the model backbone class, it is essential to check
-whether the model has been implemented exactly as the source implementation. This
-also helps in adding model "presets" at a later stage.
+- It should offer simple methods like predict(), fit(), generate() (for 
+  generative models), detect() (for detection models)
 
-The preferred way of doing this is to add a Colab link in the PR description, which
-1) converts the original preset weights to our format, and
-2) checks whether the outputs of the original model and your implemented model are close enough.
+##### Unit Tests
 
-It is okay if you demonstrate it for one preset at this stage; you can do the conversion
-for the other presets when you officially add presets to the library at a later stage.
+- Test basic usage: instantiation, forward pass with dummy data from the 
+  preprocessor, and model compilation.
 
-#### Add Unit Tests
+---
 
-It is essential to add units tests. These unit tests are basic and mostly check
-whether the forward pass goes through successfully, whether the model can be saved
-and loaded correctly, etc.
+### Step 5: PR #4 – Presets and End-to-End Validation
 
-### Step 3: PR #2 - Add XXTokenizer
+After PRs 1–3 are merged, create:
 
-#### Tokenizer
+#### Preset Configuration
 
-Most text models nowadays use subword tokenizers such as WordPiece, SentencePiece
-and BPE Tokenizer. Since KerasHub has implementations of most of the popular
-subword tokenizers, the model tokenizer layer typically inherits from a base
-tokenizer class.
+- Add `xx_presets.py`
+- Include model args, checkpoint URLs, vocabulary paths
 
-For example, DistilBERT uses the WordPiece tokenizer. So, we can introduce a new
-class, `DistilBertTokenizer`, which inherits from `keras_hub.tokenizers.WordPieceTokenizer`.
-All the underlying actual tokenization will be taken care of by the superclass.
+Use the [Kaggle org page](https://www.kaggle.com/organizations/kerashub/models) 
+to stage and test.
 
-The important thing here is adding "special tokens". Most models have
-special tokens such as beginning-of-sequence token, end-of-sequence token,
-mask token, pad token, etc. These have to be
-[added as member attributes](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/distil_bert/distil_bert_tokenizer.py#L91-L105)
-to the tokenizer class. These member attributes are then accessed by the
-preprocessor layers.
+#### `from_preset()` Functions
 
-For a full list of the tokenizers KerasHub offers, please visit
-[this link](https://keras.io/api/keras_nlp/tokenizers/) and make use of the
-tokenizer your model uses!
+Add this to:
+- `XXBackbone`
+- `XXTokenizer`
 
-#### Unit Tests
+[Example](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/distil_bert/distil_bert_backbone.py#L187-L189)
```

Comment:
You are adding a lot of outdated code links that no longer work. Please check all these!